patterns/footer-portfolio.php

```diff
@@ -76,7 +76,7 @@
 			<!-- wp:group {"style":{"spacing":{"blockGap":"6px"}},"layout":{"type":"flex","flexWrap":"wrap"}} -->
 			<div class="wp-block-group">
 				<!-- wp:paragraph {"fontSize":"small"} -->
-				<p class="has-small-font-size"><?php esc_html_e( '© 2024', 'twentytwentyfour' ); ?></p>
+				<p class="has-small-font-size"><?php esc_html_e( '&copy; 2024', 'twentytwentyfour' ); ?></p>
```

Comment:
Shall we bump this to 2023?

Reply:
There is an open [issue](https://github.com/WordPress/twentytwentyfour/issues/510) for this.Let's open a different PR for that, this one is just to bring the repo inline with what's on core right now, we can iterate on that and it's clearer when we need to move our changes to core for beta 2 if they are made outside of this PR